LoginPage.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Formik, Field, Form } from 'formik';
+import { useHistory } from 'react-router-dom';
+import './Login.css';
+
+const LoginPage = () => {
+  let history = useHistory();
+
+  const handleSubmit = () => {
+    history.push('/dashboard');
+  };
+
```

Comment:
Consider adding form validation logic to check if the user inputs for email and password are valid before submitting the form.